letsencrypt/cli.py

```diff
@@ -56,7 +56,7 @@
 the cert. Major SUBCOMMANDS are:
 
   (default) run        Obtain & install a cert in your current webserver
-  auth                 Authenticate & obtain cert, but do not install it
+  certonly             Obtain cert, but do not install it (aka "auth")
```

Comment:
UI/UX question I don't know the answer to: Do we want to include `auth` here anymore?